src/backend/fastapi_app/__init__.py

```diff
@@ -29,14 +32,15 @@ class State(TypedDict):
 
 
 @asynccontextmanager
-async def lifespan(app: FastAPI) -> AsyncIterator[State]:
+async def lifespan(app: fastapi.FastAPI) -> AsyncIterator[State]:
```

Comment:
Change necessary due to configure_azure_monitor only working with "import fastapi"